include/swift/AST/NameLookup.h

```diff
@@ -369,8 +369,8 @@ void forAllVisibleModules(const DeclContext *DC, const Fn &fn) {
 
 /// Only name lookup has gathered a set of results, perform any necessary
```

Comment:
Is there a typo? "Only" should be "Once"?

Reply:
You are right! It should be "Once". Fixed it now.